### PR TITLE
MM-841 : Remove old commons-lang lib from Gytheio

### PR DESCRIPTION
**Pull request by DenisGabriela**
- remove usage of commons-lang 2.6
   - use commons-lang3 3.9

### CODE REVIEW

File: gytheio-commons/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.gytheio</groupId>
         <artifactId>gytheio-parent</artifactId>
-        <version>0.12-SNAPSHOT</version>
+        <version>0.11.3-SNAPSHOT</version>
```
Comment (montgolfiere) [inline gytheio-commons/pom.xml]: We shouldn't go backwards. I suggest either a new branch from SP/0.11.N (and that can then be merged forward to master) *or* simply cherry-pick back to SP/0.11.N after fixing on master.

Comment (DenisGabriela) [reply gytheio-commons/pom.xml]: Makes sense, I forgot about the SP branch. Thanks.

**Comment by DenisGabriela** [comment]
I opened a separate PR on SP/0.11.N for the release.

File: gytheio-messaging/messaging-broker-activemq/src/test/webapp/admin/WEB-INF/tags/form/text.tag [hunk 2]
```diff
@@ -24,7 +24,7 @@
 	if (value == null) {
 		value = "";
 	}
-	value = org.apache.commons.lang.StringEscapeUtils.escapeHtml(value);
```
Comment (montgolfiere) [inline gytheio-messaging/messaging-broker-activemq/src/test/webapp/admin/WEB-INF/tags/form/text.tag]: Hmm - I wonder if this is correct here (where is this dependency resolved when running the copy of the ActiveMQ "admin" webapp ?

As a separate PR, I wonder if we can remove this test admin webapp (and use docker container instead) ?

File: gytheio-messaging/messaging-broker-activemq/src/test/webapp/admin/WEB-INF/tags/form/short.tag [hunk 3]
```diff
@@ -17,8 +17,8 @@
 <%@ attribute name="text" type="java.lang.String" required="true"  %>
 <%@ attribute name="length" type="java.lang.Integer" required="false" %>
 <%
- text = org.apache.commons.lang.StringEscapeUtils.escapeHtml(text);
- text = org.apache.commons.lang.StringEscapeUtils.escapeJavaScript(text);
+ text = org.apache.commons.lang3.StringEscapeUtils.escapeHtml(text);
```
Comment (montgolfiere) [inline gytheio-messaging/messaging-broker-activemq/src/test/webapp/admin/WEB-INF/tags/form/short.tag]: Hmm - I wonder if this is correct here (where is this dependency resolved when running the copy of the ActiveMQ "admin" webapp ?

As a separate PR, I wonder if we can remove this test admin webapp (and use docker container instead) ?

File: gytheio-messaging/messaging-broker-activemq/src/test/webapp/admin/WEB-INF/tags/form/text.tag (hunk 2 above)
Comment (DenisGabriela) [reply gytheio-messaging/messaging-broker-activemq/src/test/webapp/admin/WEB-INF/tags/form/text.tag]: Jira issue - https://issues.alfresco.com/jira/browse/MM-842

File: gytheio-messaging/messaging-broker-activemq/src/test/webapp/admin/WEB-INF/tags/form/short.tag (hunk 3 above)
Comment (DenisGabriela) [reply gytheio-messaging/messaging-broker-activemq/src/test/webapp/admin/WEB-INF/tags/form/short.tag]: Jira issue - https://issues.alfresco.com/jira/browse/MM-842

**Pull request merged**